website/content/docs/k8s/compatibility.mdx

```diff
@@ -9,15 +9,15 @@ description: >-
 
 For every release of Consul on Kubernetes, a Helm chart, `consul-k8s-control-plane` binary and a `consul-k8s` CLI binary is built and distributed through a single version. When deploying via Helm, the recommended best path for upgrading Consul on Kubernetes, is to upgrade using the same `consul-k8s-control-plane` version as the Helm Chart, as the Helm Chart and Control Plane binary are tightly coupled.
 
-## Supported Consul versions
+## Supported Consul and Kubernetes versions
 
 Consul Kubernetes versions all of its components (`consul-k8s` CLI, `consul-k8s-control-plane`, and Helm chart) with a single semantic version. When installing or upgrading to a specific versions, ensure that you are using the correct Consul version with the compatible `consul-k8s` helm chart and/or CLI.
 
-| Consul Version | Compatible consul-k8s Versions   |
-| -------------- | -------------------------------- |
-| 1.13.x         | 0.49.x                           |
-| 1.12.x         | 0.43.0 - 0.49.x                  |
-| 1.11.x         | 0.39.0 - 0.42.0, 0.44.0 - 0.49.x |
+| Consul Version | Compatible consul-k8s Versions   | Compatible Kubernetes Versions | 
+| -------------- | -------------------------------- | -------------------------------|
+| 1.14.x         | 1.0.x                            | 1.22.x - 1.25.x                |
+| 1.13.x         | 0.49.x                           | 1.21.x - 1.24.x                |
+| 1.12.x         | 0.43.0 - 0.49.x                  | 1.19.x - 1.22.x               |
```

Comment:
The alignment on the `1.12.x` line on the far right seems off.

Reply:
Thank you!